src/Common/Enum.cs

```diff
@@ -144,6 +144,24 @@ public enum IssueCategories
         Internal
     }
 
+    public enum ExchangeRateStates

```

Comment:
Should we rename the enum to "ExchangeRateSource" and keep only 3 values: USDOnly, File, Memory.
Default value = USDOnly. 
Call exchange api to fetch rates. If successful - update enum to Memory.
Read from json to fetch rates. If successful - update enum to File.

Reply:
Removed the concept of ExchangeRateStates completely, state is determined whether ExchangeRates dictionary is null or not.